clone.go

```diff
@@ -61,58 +62,56 @@ func Clone(c Config) error {
 		return errors.Wrapf(err, "failed to read src go.mod")
 	}
 
+	if _, err := os.Stat(filepath.Join(config.TargetDir, "go.mod")); os.IsNotExist(err) {
+		log.Println("No go.mod found in target directory, copying from source")
+		source, err := srcFilesystem.Open(filepath.Join(srcRoot, "go.mod"))
```

Comment:
Not having original go.mod was basically one of the main motivations for this project 😀

The idea is to bring as little dependencies as possible from upstream project, because the go dependency tree from upstream projects has been causing issues for us (breaking our projects).

Reply:
Good point.

Reply:
Updated to instead generate a new one based on the module and go version of the source